### PR TITLE
added new question

### PR DESCRIPTION
**Pull request by rahulsin14**

### CODE REVIEW

File: javascript/index.html [hunk 1]
```diff
@@ -119,6 +119,12 @@ <h4>
                     How would you use a closure to create a private counter?
                 </h4>
             </li>
+            <li>
+                <a href="./answers/rahulsin14_q7.html">
```
Comment (shivamyth) [inline javascript/index.html]: It should be hyphen there not an underscore

File: javascript/answers/rahulsin14_q7.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            <div>
+                <h4>
+                    Explain few difference between null, undefined or undeclared JavaScript variable?
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <div>JavaScript allows DOM elements to be nested inside each other. In such a case, if the handler of the child is clicked, the handler of parent will also work as if it were clicked too.</div>
```
Comment (shivamyth) [inline javascript/answers/rahulsin14_q7.html:51]: Also, give an example of each one.

**Review by shivamyth — CHANGES_REQUESTED** [review]
Do the changes as commented

File: javascript/answers/rahulsin14_q7.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="Define Event Bubbling">
```
Comment (arpansac) [inline javascript/answers/rahulsin14_q7.html:19]: Put question text in content=""

**Review by arpansac — CHANGES_REQUESTED** [review]
Changes requested

**Comment by dakshkhetan** [comment]
Any updates on this? This PR will be closed by tomorrow if no changes are done.

**Comment by shivamyth** [comment]
No updates in the PR, closing it.

**Pull request closed without merging**